### PR TITLE
Disable tokudb backup on --innodb-use-native-aio=on

### PR DESCRIPTION
**Pull request by vlad-lesin**
See https://tokutek.atlassian.net/browse/BACKUP-121.
http://jenkins.percona.com/job/percona-server-5.6-param/1573/

### CODE REVIEW

File: mysql-test/suite/tokudb.backup/t/tokudb_backup_disable_if_innodb_use_native_aio_enabled.test [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Check if tokudb hot backup is prevented if innodb_use_native_aio enabled
+--source include/have_tokudb_backup.inc
```
Comment (laurynas-biveinis) [inline mysql-test/suite/tokudb.backup/t/tokudb_backup_disable_if_innodb_use_native_aio_enabled.test]: --source include/have_innodb.inc

File: mysql-test/suite/tokudb.backup/t/tokudb_backup_disable_if_innodb_use_native_aio_enabled.test [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+--let BACKUP_DIR=$MYSQL_TMP_DIR/tokudb_backup
+
+--mkdir $BACKUP_DIR
+
+--replace_result $MYSQL_TMP_DIR MYSQL_TMP_DIR
+--error ER_WRONG_VALUE_FOR_VAR
+--eval SET SESSION tokudb_backup_dir='$BACKUP_DIR'
+
+--let $restart_parameters= restart:--innodb-use-native-aio=off
```
Comment (laurynas-biveinis) [inline mysql-test/suite/tokudb.backup/t/tokudb_backup_disable_if_innodb_use_native_aio_enabled.test]: Maybe not required? This is basic backup functionality, other tests in this suite already exercise it

File: mysql-test/suite/tokudb.backup/t/tokudb_backup_disable_if_innodb_use_native_aio_enabled.test [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Check if tokudb hot backup is prevented if innodb_use_native_aio enabled
```
Comment (laurynas-biveinis) [inline mysql-test/suite/tokudb.backup/t/tokudb_backup_disable_if_innodb_use_native_aio_enabled.test]: Very long test name - IMHO "innodb_native_aio.test" would be as descriptive

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: mysql-test/suite/tokudb.backup/t/tokudb_backup_disable_if_innodb_use_native_aio_enabled.test (hunk 3 above)
Comment (vlad-lesin) [reply mysql-test/suite/tokudb.backup/t/tokudb_backup_disable_if_innodb_use_native_aio_enabled.test]: done

File: mysql-test/suite/tokudb.backup/t/tokudb_backup_disable_if_innodb_use_native_aio_enabled.test (hunk 1 above)
Comment (vlad-lesin) [reply mysql-test/suite/tokudb.backup/t/tokudb_backup_disable_if_innodb_use_native_aio_enabled.test]: done

File: mysql-test/suite/tokudb.backup/t/tokudb_backup_disable_if_innodb_use_native_aio_enabled.test (hunk 2 above)
Comment (vlad-lesin) [reply mysql-test/suite/tokudb.backup/t/tokudb_backup_disable_if_innodb_use_native_aio_enabled.test]: done

**Comment by vlad-lesin** [comment]
Laurynas, could you review it once more please?

**Comment by vlad-lesin** [comment]
And 5.7 branch too.

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**